projects/account-api/Makefile

```diff
@@ -0,0 +1,3 @@
+account-api: bundle-account-api
+	$(GOVUK_DOCKER) run $@-lite bin/rails db:setup
+	$(GOVUK_DOCKER) run $@-lite env RAILS_ENV=test bin/rake db:setup
```

Comment:
Minor - we're mixing bin/rails and bin/rake here which is unnecessary. Sticking with bin/rails for both would be best.